app/window.cpp

```diff
@@ -191,6 +193,9 @@ void window::start(const std::function<void()> &body) {
 
         body();
 
+        std::clamp(maniac::config.tap_time, 0, 500);
+        std::clamp(maniac::config.humanization_modifier, 0, 1000);
```

Comment:
i don't like the arbitrary magic upper bounds on this, how about a `max(0, <input>)`?

Reply:
that would work perfectly, didnt know max existed lol